SR-IOV GA: add default route overwrite example

zshi-redhat opened the pull request.

zshi-redhat commented:
@squeed a follow-up PR for [SR-IOV GA enhancement](https://github.com/openshift/enhancements/pull/6), adding example of default route overwrite config.

squeed commented:
From my perspective, it seems that it would be more useful to be able to declare arbitrary network-gateway pairs via Multus. Something like


```
kind: Pod
metadata:
  name: my-pod
  namespace: my-namespace
  annotations:
    k8s.v1.cni.cncf.io/networks: |
      [
        { "name":"net-a" ,
          "routes": [
            { 
              "dst": "10.0.0.0/24",
              "nexthop": "192.168.0.127"
            } ]
          },
        {
          "name":"net-b",
          "routes": [
            { 
              "dst": "0.0.0.0/0",
              "nexthop": "99.99.99.99"
            } ]          
        }
      ]
```

russellb commented:
@dougbtv @zshi-redhat Hi, can you take a look at this enhancement and see if it should be rebased or just closed?  It looks like a pretty minor addition that we should be able to either update to reflect the current status and merge, or just close.

zshi-redhat commented:
> @dougbtv @zshi-redhat Hi, can you take a look at this enhancement and see if it should be rebased or just closed? It looks like a pretty minor addition that we should be able to either update to reflect the current status and merge, or just close.

Rebased and updated the PR to reflect current fact (no additional features added).

russellb commented on enhancements/network/20190903-SRIOV-GA.md:
```diff
@@ -12,19 +13,19 @@ reviewers:
-last-updated: 2019-09-03
-status: implementable
+last-updated: 2020-08-17
+status: provisional
```
Did you mean to change this from `implementable` back to `provisional` ?  Can you explain the reason?  Thanks!

zshi-redhat replied:
my bad, it was a mistake, shall be `implemented`.
updated

russellb commented:
Thanks!

/approve
/lgtm
